website/docs/i18n/i18n-crowdin.mdx

```diff
@@ -122,23 +122,16 @@ Create `crowdin.yml` in `website`:
 project_id: '123456'
 api_token_env: 'CROWDIN_PERSONAL_TOKEN'
 preserve_hierarchy: true
-files: [
-    # JSON translation files
-    {
-      source: '/i18n/en/**/*',
-      translation: '/i18n/%two_letters_code%/**/%original_file_name%',
-    },
-    # Docs Markdown files
-    {
-      source: '/docs/**/*',
-      translation: '/i18n/%two_letters_code%/docusaurus-plugin-content-docs/current/**/%original_file_name%',
-    },
-    # Blog Markdown files
-    {
-      source: '/blog/**/*',
-      translation: '/i18n/%two_letters_code%/docusaurus-plugin-content-blog/**/%original_file_name%',
-    },
-  ]
+files:
```

Comment:
👍 

We should also change our own config for consistency, as it's also part of the doc, see https://docusaurus.io/docs/next/i18n/crowdin#example-configuration

I'd rather avoid displaying distinct yaml syntax to users that would be confused